src/index.html

```diff
@@ -4,10 +4,267 @@
+          </div>
+          <div class="details__description">
+            <div class="details__heading">
+              Over 50 years experience in love.
+            </div>
+            <div class="details__paragraph">
+              <p>
+                Mazel and VA Tsukkerman in his "Analysis of musical works."<br>
+              Pointillism, which originated in the music of the early twentieth<br>
+              century, microforms, found a distant historical.
+              </p>
+
+              <p>
+                The market leader in real estate since 2003. Awards Real Estate<br>
+              Company, 2010, 2011, 2013. More than 200 satisfied customers<br>
+              premium segment. We focus only on new buildings and high-level<br>
+              doskanalno know this area better than other specialists.<br>
+              Working directly with developers - so have more useful information<br>
+              about the profitable deals.
+              </p>
+
+              <strong>We will never be late!<br>
+              Guaranteed secure transaction - 100%
+              </strong>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="section" id="gallery">
+        <h2 class="heading section__heading">Gallery<span class="heading__crown"></span></h2>
+        <div class="gallery">
+          <div class="photos photos--wide_left">
+            <img
+            src="./images/photos/1.jpg"
+            alt=""
+            class="photos__img"
+            >
+            <img
+            src="./images/photos/2.jpg"
+            alt=""
+            class="photos__img"
+            >
+          </div>
+          <p class="gallery__caption"><strong>Marina Palms</strong> / <a href="#" class="link">North Miami Beach, FL 33162</a></p>
+
+          <div class="photos">
+            <img
+            src="./images/photos/3.jpg"
+            alt=""
+            class="photos__img"
+            >
+            <img
+            src="./images/photos/4.jpg"
+            alt=""
+            class="photos__img"
+            >
+          </div>
+          <p class="gallery__caption"><strong>Edition Residents</strong> / <a href="#" class="link">Miami Beach, FL 33139</a></p>
+
+          <div class="photos photos--wide_right">
+            <img
+            src="./images/photos/5.jpg"
+            alt=""
+            class="photos__img"
+            >
+            <img
+            src="./images/photos/6.jpg"
+            alt=""
+            class="photos__img"
+            >
+          </div>
+          <p class="gallery__caption"><strong>Faena House</strong> / <a href="#" class="link">Miami Beach, FL 33140</a></p>
+
+          <button class="button gallery__button">View all</button>
+        </div>
+      </section>
+
+      <section class="section" id="consultation">
+        <h2 class="heading section__heading">Consultation<span class="heading__crown"></span></h2>
```

Comment:
```suggestion
        <h2 class="heading section__heading">
          Consultation
          <span class="heading__crown"></span>
        </h2>
```
а еще лучше воспользоваться псевдоэлементом

Reply:
Fixed all